android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -184,7 +183,8 @@ public void fetchRelease(Promise promise) {
     @ReactMethod
     public void captureEnvelope(String envelope, Promise promise) {
         try {
-            File installation = new File(sentryOptions.getOutboxPath(), UUID.randomUUID().toString());
+      String outboxPath = HubAdapter.getInstance().getOptions().getOutboxPath();
```

Comment:
```suggestion
      final String outboxPath = HubAdapter.getInstance().getOptions().getOutboxPath();
      if (outboxPath == null) {
          logger.severe("outboxPath isn't set.");
      }
```

Reply:
Let's log if outboxPath isn't set yet, that means SDK is yet not inited, the logger message could be improved.